skills_ml/storage/__init__.py

```diff
@@ -199,3 +224,82 @@ def save(self):
 
         json_bytes = json.dumps(self._storage).encode()
         self.fs.write(json_bytes, self.fname)
+
+
+class ModelStorage(object):
+    def __init__(self, storage=None):
+        self._storage = FSStore() if storage is None else storage
+
+    @property
+    def storage(self):
+        return self._storage
+
+    @storage.setter
+    def storage(self, value):
+        if hasattr(value, 'write') and hasattr(value, 'load'):
+            self._storage = value
+        else:
+            raise Exception(f"{value} should have methods 'write()' and 'load()'")
+
+    def load_model(self, model_name, **kwargs):
+        """The method to load the model from where Storage object specified
+
+        model_name (str): name of the model to be used.
+        """
+        with self.storage.open(model_name, "rb") as f:
+            model = joblib.load(f)
+
+        return model
+
+    def save_model(self, model, model_name):
+        """The method to write the model to where the Storage object specified
+
+        model_name (str): name of the model to be used.
+        """
+        with self.storage.open(model_name, "wb") as f:
+            joblib.dump(model, f, compress=True)
+
+
+class SerializableModel(object):
```

Comment:
I would like docstrings for these new classes, especially since it's not clear based on their names how some of them (for instance ProxyObjectWithStorage and SerializableModel) differ from each other in purpose.